project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject starcity/stripe-clj "0.3.5-SNAPSHOT"
+(defproject starcity/stripe-clj "0.3.6-SNAPSHOT"
```

Comment:
The project version shouldn't be changed—version bumps only happen at release time.

Reply:
Actually, I see that Version 0.3.5 has already been released now, hence the need for this change.

Reply:
So should I change version back to `0.3.5-SNAPSHOT`?

Reply:
No, never mind my previous comment.